AirSkyBoat/AirSkyBoat
# Add in missing messages in action packets for Curaga to display healing amounts.

**_I affirm:_**

## What does this pull request do?

This PR fixes curaga messages stemming from incorrect checks if spells are healing damage and adds in the missing MSGID for "<target> recovers <amount> HP" used in retail (263).

Added missing MSGBASIC_RECOVERS_HP = 263
Added isCure/isHeal check instead of checking for damage because damage value is absolute, it's not negative anymore. Added check if the action target is the caster or if they're just receiving healing so it delivers the correct msg.

## Steps to test these changes

To test this change, log in at least two accounts.  Subsequently cast curaga and observe correct messaging.
